lispify/lispify.py

```diff
@@ -9,11 +9,18 @@
 so that the lispify method ignores it.
 """
 
+from __future__ import absolute_import
+
 from numbers import Number
+from sys import version_info
 import warnings
 
-from util import subclasses, camel_case_to_lisp_name
+from lispify.util import subclasses, camel_case_to_lisp_name
+
 
+# python 3 compatibility functions
+def string_types():
```

Comment:
I am fine with 3rd party library if you're also fine with it. There are two compatibility libraries: `future` and `six`, and I'm not sure which one is easier to use.